schema/backwards_compatibility_test.go

```diff
@@ -100,8 +100,7 @@ func TestBackwardsCompatibilityImageIndex(t *testing.T) {
          "digest": "sha256:fb2fc0707b86dafa9959fe3d29e66af8787aee4d9a23581714be65db4265ad8a",
          "platform": {
             "architecture": "arm64",
-            "os": "linux",
-            "variant": "armv8"
```

Comment:
I don't think we should add it until upstream Go adds "8".

Note that there is no guarantee Go adds "8".
It can be "8-foo" depending on the future extension, or maybe completely different name depending on the future ARM versioning scheme.